workflow/VERSION

```diff
@@ -1 +1 @@
-v2.1.0
+v2.1.1
```

Comment:
What is the criteria for when these change? Should there be something?

Reply:
We just had a conversation in a meeting about trying to version each PR following this convention:
[https://github.com/NOAA-EMC/rrfs-workflow/issues/767](https://github.com/NOAA-EMC/rrfs-workflow/issues/767)

To facilitate starting this from `X.Y.Z-1.0`,  we want to bump the version number to `2.1.1`